packages/website-eslint/README.md

```diff
@@ -0,0 +1,18 @@
+# `website-eslint`
+
+A bundled version of ESLint plus `typescript-eslint`, made to work in a browser.
+This is exclusively used for the playground in the [`website` package](../website/README.md).
+
+## Building
+
+`yarn build` runs `build.ts`, which uses ESBuild to create a CommonJS bundle including:
+
+- ESLint's [`Linter` class](https://eslint.org/docs/latest/integrate/nodejs-api#linter) and built-in rules
+- TypeScript's `typescript` and `typescript/lib/tsserverlibrary` module entry points
```

Comment:
This isn't right - the website downloads the TS version on the fly. We fetch TS from the same source that the TS playground does so we get a "Monaco web" compatible bundle and so we can DL any supported version